examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -439,44 +439,20 @@ def main():
     elif accelerator.mixed_precision == "bf16":
         weight_dtype = torch.bfloat16
 
+    # Freeze the unet parameters before adding adapters
+    for param in unet.parameters():
+        param.requires_grad_(False)
+
+    unet_lora_config = LoraConfig(
+        r=args.rank, target_modules=["conv1", "conv2", "conv_shortcut", "proj_in", "proj_out"]
```

Comment:
Target modules should be either fixed or be made CLI arg IMO. 

For our training scripts with LoRA, we only focus on the attention matrices (key, query, and value).